apps/aecore/src/aec_headers.erl

```diff
@@ -190,7 +192,13 @@ type(#mic_header{}) -> micro.
 
 -spec from_db_header(tuple() | header()) -> header().
 %% We might have a legacy tuple in the db
```

Comment:
As far as I understand the PR, it would be fine to upgrade to this version, get a database with fixed flags and then downgrade. Because the flags do not mean anything for the old version?

Reply:
That's a pity...